frontend/all.js

```diff
@@ -41,7 +41,8 @@ function initAll() {
 
   var $filters = document.querySelectorAll('[data-module="activities-list-filter"]')
   nodeListForEach($filters, function ($filter) {
-    new ListFilter($filter)
+    const startShown = $filter.getAttribute('data-filter-start-shown') === 'true'
+    new ListFilter($filter, startShown)
```

Comment:
I've asked Lee to take a gander at this, see if its bundled/built as he'd expect. Not had the change to look over the bunding myself yet.

Reply:
Yep, looks good. `ListFilter` is exported below then imported by `index.js` which is the rollup bundle target.